Kentico.Kontent.Delivery/Extensions/DeliveryOptionsExtensions.cs

```diff
@@ -2,8 +2,16 @@
 
 namespace Kentico.Kontent.Delivery.Extensions
 {
-    internal static class DeliveryOptionsExtensions
+    /// <summary>
+    /// A class which contains extension methods on <see cref="DeliveryOptions"/>.
+    /// </summary>
+    public static class DeliveryOptionsExtensions
```

Comment:
shouldn't we just move this one to the abstractions lib?

Reply:
Yes, I do.